handbook/people-ops/paid-time-off-and-working-hours.md

```diff
@@ -1 +1,33 @@
-# Paid time off and working hours
\ No newline at end of file
+# Paid time off and working hours
+
+At Sourcegraph, every teammate has agency over how they create a balance between work and rest, so that we can stay productive and innovative over the long term.
+
+This balance looks different from person to person and may be reliant on a number of factors, including a teammate’s preferred working style, location, and family obligations. For that reason, we empower our teammates to have agency over their working times, the vacations they take, the time they need to recharge, and the number of hours they can put in over any week without losing focus or motivation. Here are some basic guidelines to help our teammates navigate their paid time off for rest, paid time off for reasons other than rest, and working hours.
+
+## Paid time off (for rest)
+
+Our company-wide policy is unlimited paid time off in exchange for achieving work outcomes that impact our success, but we encourage our teammates to take a minimum of 30 days total paid time off for rest in a calendar year. Your total paid time off refers to vacation days, bank/public holidays, and religious days, and you can mix these days up as they best suit you, but remember that the intention is for you to disconnect and get rest.
+
+-   Local guidance on bank/public holidays and religious days: you’re entitled to all the holidays celebrated in your country of residence and/or by your religious denomination, and it is assumed you will be taking paid time off on these days. If you choose to work one of these public holidays, you may exchange them for another day, just update your availability status accordingly. 
+-   Local guidance on vacation days: please ensure you comply with your resident country’s legislated minimum vacation days.
+
+## Paid time off (for other reasons)
+
+ -  **Wellbeing leave**: we trust you to practice agency over your wellbeing, and provide wellbeing leave to take if you are ill or need time away from work to regulate your mental or physical wellbeing. Additional wellbeing leave is also warranted if you’ve been stretched in supporting your team through growth, so that you can recalibrate.
```

Comment:
It seems a bit weird to me that vacation and sick days are in the same category. As a manager, I wouldn't want to look at a stat and think _"Great, looks like my report took 5 days of vacation/wellbeing leave this quarter"_ when the reality may be that they were sick and suffering from illness for a week, were therefor _forced_ to take that time off and otherwise took **zero** days of vacation/actual "rest"/"wellbeing leave".

It would make more sense to me to distinguish days one is forced to take (illness, life leave) and don't necessarily equate to _rest_ from those for rest/vacation, the "happy" time off, so that we can make sure everybody takes enough _happy_ time off and not inflate that number through the other category.

But maybe I'm missing something about the motivation behind the current split.

Reply:
@felixfbecker these actually are in different categories! Vacation would be considered PTO for **rest** (along with public holidays and religious holidays), these count towards the minimum 30 days.

In the second category (with no defined minimum/maximum since needs will vary) we have PTO for **wellbeing** (this is specific to sick days– any day where someone is not feeling well, whether physically, mentally, etc.).

I want to make sure that is super clear in this documentation, so it seems that it might not be. Let me know if you think any particular parts of the wording would help improve this!

Reply:
Ohh, sorry, despite having read the whole document I somehow missed that there are two higher level categories, and two sub-categories here. I thought the sub-categories were the top-level categories 🤦🏻

Reply:
No worries– I really appreciate the input you've given on our PTO rollout!